### PR TITLE
Nominator Rulechanges to account for tiering removal and slight rewording

### PR DESCRIPTION
**Pull request by Okorin**
### Description

Beatmap Nominator Rules needed an update to reflect tiering removal accordingly, i left the changelog within the article empty because it would list a lot of stuff that doesn't affect anyone because it's not new.

### Status

Discussion finished with the BNG and the QAT pretty much

### Changelog

- tiers were removed and replaced with probation period
- slight rewordings on a lot of rules to make them more simple or way easier to understand
- none of the newly added rules need a delay to come into effect so they weren't added to the changelog

### CODE REVIEW

File: wiki/People/Beatmap_Nomination_Group/Rules/en.md [hunk 1]
```diff
@@ -15,32 +15,34 @@ Violations of those rules may be reported to a member of the [Quality Assurance
...
-+ **Do not nominate a beatmap if you cannot reasonably judge it.** Being able to reasonably play a beatmap is a core part of being able to judge a beatmap’s quality. If you are multiple tiers below the playing level of the map we may call your ability to judge it into question.
-+ **Do not qualify a beatmap if anything significant on the beatmap got changed since it was bubbled.** If you want to qualify directly, make sure that the changes made were minor and that you rechecked your changes with the original bubbler. When in doubt, rebubble such beatmaps.
-+ **Do not follow any unfinalized rules or amendments.** Only act upon the set of rules that is currently in effect, the rules of tomorrow are uncertain so they should not be followed.
-+ **Make sure no mod posts in the map thread have been ignored before qualifying or bubbling a map.** Posts without a sufficient reply may count as ignored. When in doubt, double check if the Code of Conduct for Mapping and Modding has been violated on the mapthread and act accordingly.
+    - how to enforce / check this? General mindset stuff seems weird if we cannot check for it
```
Comment (Ephemeralis) [inline wiki/People/Beatmap_Nomination_Group/Rules/en.md]: remove this line probably

File: wiki/People/Beatmap_Nomination_Group/Rules/en.md [hunk 2]
```diff
@@ -15,32 +15,34 @@ Violations of those rules may be reported to a member of the [Quality Assurance
...
-+ **If a bubbled map contains unrankable issues, its bubble must be popped.** To ensure a map has no other unrankable issues, it is best for another Nominator to examine the map once again.
-+ **You must not rebubble a map when your bubble was popped without the popper agreeing to that action.** In these cases another nominator may rebubble the map if they disagree with the bubble pop.
-+ **There must be at least 24 hours between placing a bubble and qualifying a mapset.** This is to give other members of the community and Beatmap Nominators a bit of time to think about whether they want to veto the placed bubble.
-+ **Do not qualify over any manually popped bubble.** A manually popped bubble resets the bubble counter to zero, so doing this is equivalent to directly qualifying a beatmap without any prior icons.
+    - i feel dumb that we thought that was necessary given that at the time we wrote it there was a test about this stuff
```
Comment (Ephemeralis) [inline wiki/People/Beatmap_Nomination_Group/Rules/en.md]: remove this line

**Comment by Okorin** [comment]
how do i always derp at least once?

fixed!

**Comment by Ephemeralis** [comment]
its a sign of brilliance to make mistakes

that's what i tell myself every morning anyway

**Pull request merged**